### PR TITLE
Minor grammatical fixes in README.md

### PR DESCRIPTION
**Pull request by iKevinY**
I could only find [one instance](https://github.com/rust-lang/rust/blob/master/src/doc/intro.md#learning-more) of 'Rustic' in Rust's documentation, but it feels like a proper noun.

### CODE REVIEW

**Comment by steveklabnik** [comment]
It's not a noun, it's an adjective.

**Comment by iKevinY** [comment]
Ah whoops, I meant to type proper adjective; I was thinking 'Rustic' should be capitalized similarly to the way 'Pythonic' is used in Python circles. Anyways, there's a couple of other fixes in this PR if you'd like the rustic change to be reverted.

**Comment by steveklabnik** [comment]
Ahh yes, I didn't notice the others. Let's take those too.

**Comment by iKevinY** [comment]
Resolved merge conflicts and force pushed.

File: examples/README.md [hunk 1]
```diff
@@ -3,17 +3,16 @@
 [Rust][rust] is a programming language with a focus on
 type safety, memory safety, concurrency and performance.
 
-This book is a collection of bitesize examples that illustrate various rustic
+This book is a collection of bitesize examples that illustrate various Rustic
```
Comment (steveklabnik) [inline examples/README.md]: still `R`.

Comment (iKevinY) [inline examples/README.md]: Whoops, fixing now.

**Comment by steveklabnik** [comment]
Thanks! :D

**Pull request merged**